functions/internal/util/util.go

```diff
@@ -29,6 +31,9 @@ type Context struct {
 // *http.Request.
 func NewContext(w http.ResponseWriter, r *http.Request) (Context, StatusError) {
 	ctx := r.Context()
+	
+	AddHSTS(w)
```

Comment:
Let's move this to `MakeHTTPHandler` so that all HTTP-related preprocessing happens there.